SPECS/perl-Crypt-Random-Seed/perl-Crypt-Random-Seed.spec

```diff
@@ -0,0 +1,146 @@
+Summary:        Simple method to get strong randomness
+Name:           perl-Crypt-Random-Seed
+Version:        0.03
```

Comment:
last change to that perl code is from 2013

-> https://metacpan.org/dist/Crypt-Random-Seed
-> https://github.com/danaj/Crypt-Random-Seed